### PR TITLE
Tensorflow pr 1.3

### PR DESCRIPTION
**Pull request by nvarini**

### CODE REVIEW

File: var/spack/repos/builtin/packages/tensorflow/package.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from spack import *
+
+
+class Tensorflow(Package):
+    """Tensorflow external package
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/tensorflow/package.py]: This is what we'll be writing in module files. Can you put a more sensible description? Ideally it should be a short and sweat first paragraph (will be module whatis) followed by a broader explanation (module help).

File: var/spack/repos/builtin/packages/tensorflow/package.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    """
+
+    homepage = "https://www.tensorflow.org/"
+    url = 'fakeurl.tar.gz'
+    licensed = True
+    only_binary = True
+
+    version('1.3')
+    depends_on('python')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/tensorflow/package.py]: As this is a fake package, please remove every `depends_on` (they don't make sense as we don't build the package with Spack).

File: var/spack/repos/builtin/packages/tensorflow/package.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    depends_on('cuda')
+    depends_on('cudnn')
+    depends_on('mpi')
+
+    def install(self, spec, prefix):
+        pass
+
+    def setup_environment(self, spack_env, run_env):
+        venv_path = '/ssoft/spack/external/tensorflow/\
+                     1.3.0/tensorflow_venv_gpu/'
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/tensorflow/package.py]: :astonished:

**Review by alalazo — CHANGES_REQUESTED** [review]
This package needs some clean-up, in particular the docstring which will be used to generate module files.

File: var/spack/repos/builtin/packages/tensorflow/package.py (hunk 1 above)
Comment (nvarini) [reply var/spack/repos/builtin/packages/tensorflow/package.py]: Ok, I just followed the description for the tensorflow packaged deployed with lafnestcha

File: var/spack/repos/builtin/packages/tensorflow/package.py (hunk 3 above)
Comment (nvarini) [reply var/spack/repos/builtin/packages/tensorflow/package.py]: ??

**Comment by alalazo** [comment]
Closing this as inactive for a while. Feel free to reopen if you have a recipe for tensorflow

**Pull request closed without merging**